phylogenetic/rules/annotate_phylogeny.smk

```diff
@@ -1,32 +1,41 @@
 """
 This part of the workflow creates additonal annotations for the phylogenetic tree.
 
-REQUIRED INPUTS:
-
-    metadata            = data/metadata.tsv
-    prepared_sequences  = results/prepared_sequences.fasta
-    tree                = results/tree.nwk
-
-OUTPUTS:
-
-    node_data = results/*.json
-
-    There are no required outputs for this part of the workflow as it depends
-    on which annotations are created. All outputs are expected to be node data
-    JSON files that can be fed into `augur export`.
-
-    See Nextstrain's data format docs for more details on node data JSONs:
-    https://docs.nextstrain.org/page/reference/data-formats.html
-
-This part of the workflow usually includes the following steps:
-
-    - augur traits
-    - augur ancestral
-    - augur translate
-    - augur clades
-
 See Augur's usage docs for these commands for more details.
 
-Custom node data files can also be produced by build-specific scripts in addition
-to the ones produced by Augur commands.
 """
+
+rule ancestral:
```

Comment:
consider `log` and `benchmark` for these?

Reply:
+1 for use of `log` and `benchmark` for nicer debugging and benchmarking experiences. 

The [ingest curate rule](https://github.com/nextstrain/rabies/blob/ac1a4780d4ececdec5e885126755b1f9e7f6fcfd/ingest/rules/curate.smk#L66-L69) is a good example that uses both.

Reply:
Done in [09932f9](https://github.com/nextstrain/rabies/pull/3/commits/09932f9ca255681e1851e2730f85c505a7939572)